### PR TITLE
Trp 7

### PR DESCRIPTION
**Pull request by StimkaT**

### CODE REVIEW

File: projects/app/src/lib/+state/commands-list/commands-list.reducer.ts [hunk 1]
```diff
@@ -20,12 +20,13 @@ export const initialState: CommandsListState = {
     commandsList: [{
         name: 'Connect to device', 
         description: '{"event":"CONNECT_USB_DEVICE","data":{"name":"COM123","timestamp":000}}',
-        message: {event:"CONNECT_USB_DEVICE",data:{name:"COM123",timestamp:111}}
+        message: {event:"CONNECT_USB_DEVICE",data:{deviceName:"COM123",timestamp:111}}
+
```
Comment (StimkaT) [inline projects/app/src/lib/+state/commands-list/commands-list.reducer.ts:24]: empty line

File: projects/app/src/lib/+state/commands-list/commands-list.reducer.ts [hunk 2]
```diff
@@ -20,12 +20,13 @@ export const initialState: CommandsListState = {
     commandsList: [{
         name: 'Connect to device', 
         description: '{"event":"CONNECT_USB_DEVICE","data":{"name":"COM123","timestamp":000}}',
-        message: {event:"CONNECT_USB_DEVICE",data:{name:"COM123",timestamp:111}}
```
Comment (StimkaT) [inline projects/app/src/lib/+state/commands-list/commands-list.reducer.ts:23]: add space

File: projects/app/src/lib/+state/commands-list/commands-list.reducer.ts [hunk 3]
```diff
@@ -20,12 +20,13 @@ export const initialState: CommandsListState = {
...
+        message: {event:"CONNECT_USB_DEVICE",data:{deviceName:"COM123",timestamp:111}}
+        
     }, 
     {
         name: 'Disconnect to device', 
         description: '{"event":"DISCONNECT_USB_DEVICE","data":{"name":"COM123","timestamp":000}}',
-        message: {event:"DISCONNECT_USB_DEVICE",data:{name:"COM123",timestamp:111}}
+        message: {event:"DISCONNECT_USB_DEVICE", data:{deviceName:"COM123",timestamp:111}}
```
Comment (StimkaT) [inline projects/app/src/lib/+state/commands-list/commands-list.reducer.ts:29]: add space

File: projects/blink/src/lib/+state/blink-commands-list/blink-commands-list.reducer.ts [hunk 4]
```diff
@@ -13,9 +13,9 @@ export interface BlinkCommandsListPartialState {
 
 export const initialState: BlinkCommandsListState = {
     commands: [
-    {name: 'ON Indicator', description:'{command:"ON"}', message: {event:"LED",data:{"command":"ON","timestamp":"1707740012056"}}}, 
-    {name: 'OFF Indicator', description:'{command:"OFF"}', message: {event:"LED",data:{"command":"OFF","timestamp":"1707740012056"}}},
-    {name: 'BLINK Indicator', description:'{command:"BLINK"}', message: {event:"LED",data:{"command":"BLINK","timestamp":"1707740012056"}}}
+    {name: 'ON Indicator', description:'{event:"LED",data:{"command":"ON","timestamp":"1707740012056"}}', message: {event:"LED",data:{"command":"ON","timestamp":"1707740012056"}}},
```
Comment (StimkaT) [inline projects/blink/src/lib/+state/blink-commands-list/blink-commands-list.reducer.ts:16]: move to new line

File: projects/tripod/src/lib/+state/messages/messages.actions.ts [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import { createAction, props } from '@ngrx/store';
+import { ILog, IMessage } from './messages.reducer';
+
+export const setDeviceName = createAction(
+  `[blinkMessages] setDeviceName`,
```
Comment (StimkaT) [inline projects/tripod/src/lib/+state/messages/messages.actions.ts:5]: rename blinkMessages to 'deviceMessages' or 'messages'

File: projects/tripod/src/lib/+state/messages/messages.actions.ts [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+  props<{ deviceName: string }>()
+);
+
+export const sendMessageToDevice = createAction(
+  `[blinkMessages] sendMessageToDevice`,
```
Comment (StimkaT) [inline projects/tripod/src/lib/+state/messages/messages.actions.ts:10]: rename blinkMessages to deviceMessages

File: projects/tripod/src/lib/+state/messages/messages.actions.ts [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+  props<{ message: IMessage }>()
+);
+
+export const setLog = createAction(
+  '[blinkMessages] setLog',
```
Comment (StimkaT) [inline projects/tripod/src/lib/+state/messages/messages.actions.ts:15]: rename blinkMessages to deviceMessages

File: projects/tripod/src/lib/+state/messages/messages.actions.ts [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+  props<{ log: ILog }>()
+);
+
+export const messageForWidget = createAction(
+  `[blinkMessages] messageForWidget`,
```
Comment (StimkaT) [inline projects/tripod/src/lib/+state/messages/messages.actions.ts:20]: rename blinkMessages to deviceMessages

**Review by sislex — APPROVED** [review]

**Pull request merged**